### DISCUSSION TITLE
Landing page

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/opendatasicilia.it#20] [Idee/Proposte]
Sera! Nonostante il nostro sia un blog, possiamo pensare di realizzare anche una landing page (un minimo accattivante) in modo tale che chiunque non sappia cosa sia ods, non appena apre il sito, si rende conto di chi siamo? Sono consapevole dell'esistenza della sezione "Chi siamo" ma vorrei evitare che all'apertura del sito, il/la visitatore/rice veda come prima (ed unica cosa) i blog-post. Che ne pensate? Posso procedere a realizzare una bozza di landing page?

**Comment by aborruso** [comment]
Certo, vai pure
